examples/app/BUILD.bazel

```diff
@@ -20,18 +22,47 @@ ts_devserver(
 
 rollup_bundle(
     name = "bundle",
-    enable_code_splitting = False,
     entry_point = ":app.ts",
     deps = [":app"],
 )
 
+babel(
```

Comment:
seems like we shouldn't try to do differential loading in every example, if that's not a built-in feature anymore

Reply:
hmmm. maybe add ts_library to the differential_loading macro in examples/webapp instead?

Reply:
agreed. I'll put ts_library into examples/webapp so it shows the full path from ts_library -> rollup_bundle -> babel -> terser -> serve